Library/Formula/ddar.rb

```diff
@@ -9,7 +9,7 @@ class Ddar < Formula
 
   depends_on "xmltoman" => :build
   depends_on :python if MacOS.version <= :snow_leopard
-  depends_on "protobuf" => "with-python"
+  depends_on "protobuf"
```

Comment:
I can reproduce that `ImportError: No module named synctus.ddar_pb2` on some, but not all, of my machines. Don't know what's different about them; will look in to it.